packages/scope-sdk/src/constants/ScopePairs.ts

```diff
@@ -83,5 +83,9 @@ export const ScopePairs = [
   'USDCet/USD',
   'HNT/USD',
   'HNTEma/USD',
+  'MOBILE/HNT',
+  'MOBILETwap/HNT',
+  'IOT/HNT',
+  'IOTTwap/HNT',
 ] as const;
-export type ScopePair = (typeof ScopePairs)[number];
+export type ScopePair = typeof ScopePairs[number];
```

Comment:
i think you have an older version of the prettier or the pipeline has an old one